### PR TITLE
RA-1806:Double Brace Initialization should not be used

### PR DESCRIPTION
**Pull request by irenesam**
https://issues.openmrs.org/browse/RA-1806

### CODE REVIEW

**Comment by irenesam** [comment]
https://issues.openmrs.org/browse/RA-1806

**Comment by irenesam** [comment]
https://issues.openmrs.org/browse/RA-1806

**Comment by HerbertYiga** [comment]
@irenesam  thanks,you can squash you commits too

**Comment by dkayiwa** [comment]
@HerbertYiga did you take a look at this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

File: omod/pom.xml [hunk 1]
```diff
@@ -220,24 +220,7 @@
                 </executions>
             </plugin>
 
-            <plugin>
-                <groupId>de.saumya.mojo</groupId>
-                <artifactId>gem-maven-plugin</artifactId>
```
Comment (dkayiwa) [inline omod/pom.xml:225]: Is this change needed for the ticket?

File: omod/pom.xml [hunk 2]
```diff
@@ -247,23 +230,7 @@
             <id>watch-sass</id>
             <build>
                 <plugins>
-                    <plugin>
-                        <groupId>de.saumya.mojo</groupId>
-                        <artifactId>gem-maven-plugin</artifactId>
-                        <extensions>true</extensions>
```
Comment (dkayiwa) [inline omod/pom.xml:253]: Is this change needed for the ticket?

File: omod/pom.xml (hunk 1 above)
Comment (irenesam) [reply omod/pom.xml:225]: Thanks @dkayiwa. Without making that change I could not run maven without errors.

Comment (dkayiwa) [reply omod/pom.xml:225]: Can you revert those changes and i see the errors that you get?

**Comment by Wandji69** [comment]
@irenesam can you update the ticket, It shows unassign meanwhile you have opened a PR for it.
This isn't good since someone else could pick it up thinking it's a free ticket.

**Comment by HerbertYiga** [comment]
> This isn't good since someone else could pick it up thinking it's a free ticket.

she might have forgotten since she is just a new comer,

File: omod/pom.xml (hunk 1 above)
Comment (irenesam) [reply omod/pom.xml:225]: Thanks @dkayiwa . The errors I get are here https://snipboard.io/F83AoR.jpg with the full log here   https://pastebin.com/b2Mp9wrA

**Comment by irenesam** [comment]
Thanks @Wandji69 and @HerbertYiga. I am sorry about that. I have rectified it.
@HerbertYiga, tomorrow at 3pm(Kampala time) will work for me. I will be on IRC first. Thanks for your help

File: omod/pom.xml (hunk 1 above)
Comment (dkayiwa) [reply omod/pom.xml:225]: Can you make a follow up commit with those changes reverted and we see?

**Comment by Wandji69** [comment]
> The changes in the pom.xml file have now been reverted

Once you revert the changes, try to build the project so you see if the build was successful.

**Comment by irenesam** [comment]
Thanks @Wandji69 . These are the errors I now get  https://pastebin.com/8bLxiEH4

**Comment by Wandji69** [comment]
> Thanks @Wandji69 . These are the errors I now get https://pastebin.com/8bLxiEH4

@irenesam I have created a GitHub gist [here](https://gist.github.com/Wandji69/7180202769885781492bffbc495ab641l) for a better discussion on the log. take a look at the logs I have pasted there.

**Comment by Wandji69** [comment]
I suggest you should do ```git pull upstream master``` to set your pom files up to date with the master.

**Comment by irenesam** [comment]
Thanks @Wandji69 .I have not been able to access the page from the link you pasted for the discussion.

**Comment by Wandji69** [comment]
> Thanks @Wandji69 .I have not been able to access the page from the link you pasted for the discussion.

here https://gist.github.com/Wandji69/7180202769885781492bffbc495ab641

**Comment by irenesam** [comment]
I need to work on my computer . Another person is free to work on the ticket
